src/profile/v100/address/non_fungible_resource_address.rs

```diff
@@ -43,6 +43,11 @@ macro_rules! decl_specialized_address {
                 $base_addr::try_from_bech32(&bech32).and_then(TryInto::<$specialized_address_type>::try_into)
             }
 
+            /// Returns the base address of this specialized address.
+            #[uniffi::export]
+            pub fn [< $specialized_address_type:snake _as_ $base_addr:snake>](address: &$specialized_address_type) -> $base_addr {
```

Comment:
New: `non_fungible_resource_address_as_resource_address` converting to `ResourceAddress` from `NonFungibleResourceAddress`